docs/DesigningProcesses.md

```diff
@@ -126,7 +126,16 @@ Variable once defined cannot be overwritten.
 
 ![split graph](img/variable_window.png)
   
-  
+## Map Variable 
+
+Evaluates `Expression` and assigns to `MapVariable.FieldName`. Allows
```

Comment:
Maybe better would be something like this: Defines new variable with one or more fields. Each field value is evaluated using `Expression`.

Reply:
Changed